tempodb/backend/v1/v1.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package backend.v1;
+
+import "google/protobuf/timestamp.proto";
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option go_package = "tempodb/backend";
+
+message BlockMeta {
+    string version = 1[(gogoproto.jsontag) = "format"];
+    bytes block_id = 2[(gogoproto.jsontag) = "blockID", (gogoproto.customname) = "BlockID", (gogoproto.customtype) = "UUID", (gogoproto.nullable) = false];
+    string tenant_id = 5[(gogoproto.jsontag) = "tenantID", (gogoproto.customname) = "TenantID"];
+    google.protobuf.Timestamp start_time = 6[(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.jsontag) = "startTime"];
+    google.protobuf.Timestamp end_time = 7[(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.jsontag) = "endTime"];
+    int64 total_objects = 8[(gogoproto.jsontag) = "totalObjects"];
+    uint64 size = 9;
+    uint32 compaction_level = 10[(gogoproto.jsontag) = "compactionLevel"];
+    bytes encoding = 11[(gogoproto.customtype) = "Encoding", (gogoproto.nullable) = false];
+    uint32 index_page_size = 12[(gogoproto.jsontag) = "indexPageSize"];
+    uint32 total_records = 13[(gogoproto.jsontag) = "totalRecords"];
+    string data_encoding = 14[(gogoproto.jsontag) = "dataEncoding"];
+    uint32 bloom_shard_count = 15[(gogoproto.jsontag) = "bloomShards"];
+    uint32 footer_size = 16[(gogoproto.jsontag) = "footerSize"];
+    bytes dedicated_columns = 17 [(gogoproto.customtype) = "DedicatedColumns", (gogoproto.jsontag) = "dedicatedColumns,omitempty", (gogoproto.nullable) = false];
+    // repeated bytes dedicated_columns = 17 [(gogoproto.customtype) = "DedicatedColumn", (gogoproto.jsontag) = "dedicatedColumns,omitempty", (gogoproto.nullable) = false];
+    uint32 replication_factor = 18[(gogoproto.jsontag) = "replicationFactor,omitempty"];
+}
+
+message CompactedBlockMeta {
+    BlockMeta block_meta = 1[(gogoproto.embed) = true, (gogoproto.nullable) = false, (gogoproto.jsontag) = ""];
+    google.protobuf.Timestamp compacted_time = 2[(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.jsontag) = "compactedTime"];
+}
+
+message TenantIndex {
+    google.protobuf.Timestamp created_at = 1[(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.jsontag) = "createdAt"];
+    repeated BlockMeta meta = 2;
```

Comment:
These fields should probably be pluralized `metas` and `compacted_metas` ?

Reply:
I noticed this also, but kept the name to be consistent with the original object.  I agree though, so pushed a commit for this.